php-practice_1.php

```diff
@@ -1,41 +1,115 @@
 <?php
 // Q1 変数と文字列
-
+$name = '廣川';
+echo '私の名前は「'.$name.'」です。' . "\n";
 
 // Q2 四則演算
-
+$num = 5*4;
+echo $num . "\n";
+echo ($num / 2) . "\n";
 
 // Q3 日付操作
-
+echo '現在時刻は' . date('y年m月d日 h時i分s秒') . 'です。' . "\n";
 
 // Q4 条件分岐-1 if文
-
+$device = 'windows';
+if ($device == 'windows') {
+    echo '使用OSは、windowsです。' . "\n";
+} else {
+    if ($device == 'mac') {
+        echo '使用OSは、macです。' . "\n";
+    } else {
+        echo 'どちらでもありません。' . "\n";
+    }
+}
```

Comment:
変数$deviceがmacかwindowsの場合、
「使用OSは、〇〇です。」というように〇〇の部分以外は同じ処理ですよね？

〇〇の部分は変数の値と一緒なので、下記のように変数と文字列結合を行うようにしておけば、
記述量を少なくできます！

```suggestion
$device = 'windows';
if ($device == 'windows' || $device == 'mac') {
    echo '使用OSは、' . $device . 'です。' . "\n";
} else {
    echo 'どちらでもありません。' . "\n";
}
```